components/NavigationDrawer/NavigationDrawerListItem/index.tsx

```diff
@@ -1,30 +1,31 @@
 import ListItemText from '@mui/material/ListItemText'
 import ListItemIcon from '@mui/material/ListItemIcon'
 import ListItemButton from '@mui/material/ListItemButton'
+import Link from 'next/link'
 // props for the list
 interface ListProps {
   text: string
   icon?: React.ReactNode
   collapsable?: boolean
+  route: string
 }
 
 function NavigationDrawerListItem({
   collapsable = false,
   icon,
   text,
+  route,
 }: ListProps) {
   return collapsable ? (
-    <div>
-      <ListItemButton>
-        <ListItemIcon>{icon}</ListItemIcon>
-        <ListItemText
-          primary={text}
-          primaryTypographyProps={{ variant: 'body2' }}
-        />
-      </ListItemButton>
-    </div>
-  ) : (
     <ListItemButton>
+      <ListItemIcon>{icon}</ListItemIcon>
+      <ListItemText
+        primary={text}
+        primaryTypographyProps={{ variant: 'body2' }}
+      />
+    </ListItemButton>
+  ) : (
+    <ListItemButton component={Link} href={route}>
```

Comment:
add a prop here called `selected` that lets us know what page is selected, heres how this should be implemented:

```jsx
<ListItemButton component={Link} href={route} selected={router.pathname === route}> ... </ListItemButton>
```

youll need to ensure that you add the `useRouter` in the comment above to get access to the `router` object.